src/engine-actions/create-wallet.js

```diff
@@ -3,25 +3,57 @@ const {
   initWallet
 } = require('../lnd-actions')
 
+/**
+ * Wallet recovery window number in blocks
+ * @constant
+ * @type {number}
+ * @default
+ */
+const RECOVERY_WINDOW_DEFAULT = 5000
+
+/**
+ * @param {Array<string>} seed
+ * @returns {boolean}
+ */
+async function isValidSeed (seed) {
+  if (!Array.isArray(seed)) return false
+  if (seed.length !== 24) return false
+  return true
+}
+
 /**
  * Creates a wallet
  *
  * @param {string} password - wallet password, used to unlock lnd wallet
+ * @param {Object} opts
+ * @param {Array<string>} opts.seed - 24 word cipher seed mnemonic
+ * @param {Buffer} opts.backup - binary string of an lnd backup file
```

Comment:
Is this a buffer or a string? Your comment is ambiguous

Reply:
it's a buffer (the file should already be open and sent). I didnt want to use `fs` in this function